main.go

```diff
@@ -391,8 +422,19 @@ func doMain(args []string) error {
 
 	DeploymentLogger = NewDeploymentLogManager(*runOptions.dataStore)
 
+	return handleCLIOptions(runOptions, env, device, config)
+}
+
+func handleCLIOptions(runOptions runOptionsType, env *uBootEnv, device *device, config *menderConfig) error {
+
 	switch {
 
+	case *runOptions.version:
+		ShowVersion()
+		return nil
+	case *runOptions.showArtifact:
+		return PrintArtifactName("/etc/mender/artifact_info")
```

Comment:
Oh, can you use `getConfDirPath()` here instead of hardcoding `/etc/mender`? Sorry, I forgot about it earlier.

Reply:
sounds reasonable :) fixed!